### ISSUE TITLE
New website format for SG / scraper needs updating

### ISSUE THREAD

**Issue by baryluk** [openZH/covid_19#381]
As of now, 2020-04-02 morning, https://www.sg.ch/tools/informationen-coronavirus.html has a new format:

```html
		<h4>2. April 2020</h4>
		
			
			
				<table id="sgch_accordion_list__sgch_accordion_sgch_table" class="table small-padding table-bordered" style="width: 100%">
<thead><tr class="odd" ><th>Anzahl Fälle: </th>
<th>455</th>
</tr></thead><tbody><tr class="even" ><td>Anzahl Hospitalisierte: </td>
<td>(kann noch nicht geliefert werden)</td>
</tr><tr class="odd" ><td>Anzahl Kritische (intensiv Pat.):</td>
<td>(kann noch nicht geliefert werden)</td>
</tr><tr class="even" ><td>Anzahl Entlassene: </td>
<td>(kann noch nicht geliefert werden)</td>
</tr><tr class="odd" ><td>Anzahl Verstorbene: </td>
<td>8</td>
</tr></tbody></table>
```

Current scraper expects different format, but this one is also easy to work with.

**Comment by metaodi** [comment]
This is basically the same as LU, see their scraper here: https://github.com/openZH/covid_19/blob/master/scrapers/scrape_lu.sh

**Comment by jb3-2** [comment]
Potentially fixed with https://github.com/openZH/covid_19/pull/384

**Comment by piotrpiotrowski** [comment]
I was also working on this fix and I noticed that the scraper failed [recently](https://github.com/openZH/covid_19/runs/555010371?check_suite_focus=true), we can try this #386

**Comment by jb3-2** [comment]
Seems to be fixed now with through #384
